internal/pkg/service/appsproxy/appconfig/api.go

```diff
@@ -43,6 +43,22 @@ type AuthRule struct {
 	Auth  []string `json:"auth"`
 }
 
+type PatchBody struct {
```

Comment:
`PatchBody` -> `NotifyPayload/NotifyBody`?

Reply:
Aren't these 2 different operations that currently have the same interface?

I would prefer 2, although now the same structures. 

Someone goes to add a field to the `notification` payload, and doesn't notice that this also changes the wakeup.

Reply:
:+1: